content/applications/finance/payment_acquirers/ogone.rst

```diff
@@ -5,84 +5,109 @@ Ogone
-
-.. _ogone/sha_key_out:
-
-SHA Key OUT
-~~~~~~~~~~~
+Ogone must now be configured to accept payments from Odoo. Head to :menuselection:`Configuration -->
+Technical Information --> Global Security Parameters`, select **SHA-1** as **Hash Algorithm** and
+**UTF-8** as **character encoding**. Then, go to the **Data and Origin Information** tab of the same
+page and leave the URL field of the **e-Commerce and Alias Gateway** section blank.
 
-In order to retrieve the SHA Key OUT, log into your ogone account, go to
-:menuselection:`Configuration --> Technical Information --> Transaction feedback --> All transaction
-submission modes`, and get or generate your **API Key** and **Client Key**. Be careful to copy your
-API key as you'll not be allowed to get it later without generating a new one.
+You are now required to generate a **SHA-IN** passphrase. **SHA-IN** and **SHA-OUT** passphrases are
+used to digitally sign the transaction requests and responses between Odoo and Ogone. By using these
+secret passphrases and the **SHA-1** algorithm, both systems can ensure that the information they
+receive from the other was not altered or tampered with.
 
 .. important::
-   If you are trying Ogone as a test, with the Test Account, change the **State** to *Test Mode*. We
-   recommend doing this on a test Odoo database, rather than on your main database.
-
-Configuration on Ogone
-======================
+   - If you need to use another algorithm, such as **SHA-256** or **SHA-512**, within Odoo, activate
+     the :ref:`developer mode <developer-mode>` and go to :menuselection:`General Settings -->
+     Technical --> System Parameters`. From here, search for `payment_ogone.hash.function` and
+     change the value line to the desired algorithm (`SHA-256` or `SHA-512`).
+
+Your **SHA-IN** and **SHA-OUT** passphrases should be different, and between 16 and 32 characters
+long. Make sure to use the same **SHA-IN** and **SHA-OUT** passphrases throughout the entire Ogone
+configuration, as Odoo only allows a single **SHA-IN** and single **SHA-OUT** passphrase. In the
+**DirectLink and Batch** section of the screen, add the same **SHA-IN** passphrase as the previous
+field. You can leave the IP address field blank.
+
+When done, head to :menuselection:`Configuration --> Technical Information --> Transaction Feedback`
+and check the following options:
+
+- **I would like to receive transaction feedback parameters on the redirection URLs:** should be
+  checked.
+- **Direct HTTP server-to-server request:** should to be set to *Online*.
+- Both **URL** fields should contain the same following URL:
+  `https://<your-odoo-database-url>/payment/ogone/accept`.
+
+.. image:: ogone/ogone-urls.png
+   :align: center
+   :alt: Enter the aforementioned URL in both fields.
+
+- **Dynamic eCommerce Parameters** should contain contain the following values:
+  `ALIAS, AMOUNT, CARDNO, CN, CURRENCY, IP, NCERROR, ORDERID, PAYID, PM, STATUS, TRXDATE`. Other
```

Comment:
`ALIAS, AMOUNT, CARDNO, CN, CURRENCY, IP, NCERROR, ORDERID, PAYID, PM, STATUS, TRXDATE`
--> is a single value, but your text indicates multiple values.
If you separate multiple values with a comma, then it should be
`ALIAS`, `AMOUNT`, `CARDNO`,...